Formula/neko.rb

```diff
@@ -1,11 +1,8 @@
 class Neko < Formula
   desc "High-level, dynamically typed programming language"
   homepage "http://nekovm.org"
-  # revision includes recent parameterized build targets for mac.  Use a :tag
-  # on the next release
-  url "https://github.com/HaxeFoundation/neko.git", :revision => "22c49a89b56b9f106d7162710102e9475227e882"
-  version "2.0.0-22c49a8"
-  revision 2
+  url "https://github.com/HaxeFoundation/neko.git", :tag => "v2-1-0", :revision => "63c8ff1a20e72e295456a1b3083209efec1e8ab2"
```

Comment:
Any reason not to use the GitHub release tarball <https://github.com/HaxeFoundation/neko/archive/v2-1-0.tar.gz> instead? We prefer that for tagged releases instead of making every user do a Git clone.